docs/ballerina-by-example/examples/base-path-and-path/base-path-and-path.description

```diff
@@ -1,3 +1,3 @@
-// Ballerina supports writing RESTful services according to the JAX-RS specification. BasePath,
-// Path, and HTTP verb annotations such as POST, GET, etc. can be used to
-// constrain your service in RESTful manner.
+// Ballerina supports writing RESTful services according to the JAX-RS specification. 
+// You can use BasePath, Path, and HTTP verb annotations such as POST and GET to
```

Comment:
Shouldn't we say POST, GET, etc.? Since there are other verbs such as PUT, DELETE, HEAD, ...

Reply:
The term "such as" usually means that only some members of a given list will be mentioned. The list consists of all HTTP methods, but we are listing only 2 as an example. We don't use etc. with that.